libcentrifugo/engineredis.go

```diff
@@ -173,11 +174,29 @@ func NewRedisEngine(app *Application, conf *RedisEngineConfig) *RedisEngine {
 
 	pool := newPool(server, password, db, conf.PoolSize)
 
+	pubScriptSource := `
```

Comment:
Perhaps add a comment here explaining what parameters are expected to mean? i.e. what are `KEYS[1|2]` and `ARGV[1..4]`?